CMakeLists.txt

```diff
@@ -68,6 +68,11 @@ ament_export_dependencies("rcl" "rcpputils" "rosidl_default_runtime" "statistics
 if(BUILD_TESTING)
   find_package(ament_lint_auto REQUIRED)
   find_package(ament_cmake_gtest REQUIRED)
+  find_package(performance_test_fixture REQUIRED)
+
+  # Give cppcheck hints about macro definitions coming from outside this package
+  get_target_property(ament_cmake_cppcheck_ADDITIONAL_INCLUDE_DIRS
+    performance_test_fixture::performance_test_fixture INTERFACE_INCLUDE_DIRECTORIES)
```

Comment:
You shouldn't need both this and the `unknownMacro` suppression in the code. I don't care which approach is used, but I don't think both are appropriate.